server/controllers/auth.js

```diff
@@ -52,33 +52,6 @@ exports.registerUser = asyncHandler(async (req, res, next) => {
   }
 });
 
-// @route POST /auth/demoLogin
-// @desc Login a demo user
-// @access Public
-exports.loginDemoUser = asyncHandler(async (req, res, next) => {
-  const { email } = req.body;
-  const id = "dogLover1234";
-  const username = "doggieBoy";
-  const token = generateToken(id);
-
-  const secondsInWeek = 604800;
-
-    res.cookie("token", token, {
-      httpOnly: true,
-      maxAge: secondsInWeek * 1000
-    });
-
-    res.status(200).json({
-      success: {
-        user: {
-        id: id,
-        username: username,
-        email: email
-      }
-    }
-  });
-})
-
```

Comment:
Just a tip for the future, it might be better to separate these changes into its own PR.

Reply:
yes, ugur, I was planning to merge the hotfix directly into the main but then I remembered that all changes need to be approved, so I just decided to merge it into this PR for you to review ...